### PR TITLE
Bump version to 2.0a0, fix setup.py, add docs

### PR DESCRIPTION
**Pull request by discordianfish**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -26,37 +14,36 @@
...
-        )
+    name='diambraArena',
+    url='https://github.com/diambra/diambraArena',
+    version='2.0a0',
+    author="DIAMBRA Team",
+    author_email="info@diambra.ai",
+    description="DIAMBRA™ Arena. Built with OpenAI Gym Python interface, easy to use,\ntransforms popular video games into Reinforcement Learning environments",
+    long_description = (Path(__file__).parent / "README.md").read_text(),
+    long_description_content_type="text/markdown",
+    license='GNU Affero GPL',
```
Comment (alexpalms) [inline setup.py:31]: also this needs to be changed, not sure what's best to put in here

File: setup.py [hunk 2]
```diff
@@ -26,37 +14,36 @@
...
-        packages=[package for package in find_packages() if package.startswith("diambraArena")],
-        include_package_data=True,
-        extras_require=extras
-        )
+    name='diambraArena',
+    url='https://github.com/diambra/diambraArena',
+    version='2.0a0',
+    author="DIAMBRA Team",
+    author_email="info@diambra.ai",
+    description="DIAMBRA™ Arena. Built with OpenAI Gym Python interface, easy to use,\ntransforms popular video games into Reinforcement Learning environments",
```
Comment (alexpalms) [inline setup.py:28]: Let's change this to "DIAMBRA™ Arena. Built with OpenAI Gym, easy to use, transforms popular video games into Reinforcement Learning environments." removing the \n new line

File: setup.py [hunk 3]
```diff
@@ -26,37 +14,36 @@
...
-            'opencv-python>=4.4.0.42',
-            'grpcio',
-            'grpcio-tools'],
-        #packages=['diambraArena','diambraArena/wrappers','diambraArena/utils'],
-        packages=[package for package in find_packages() if package.startswith("diambraArena")],
-        include_package_data=True,
-        extras_require=extras
-        )
+    name='diambraArena',
+    url='https://github.com/diambra/diambraArena',
```
Comment (alexpalms) [inline setup.py:24]: Can we also add the link to diambra website https://diambra.ai?

File: setup.py [hunk 4]
```diff
@@ -26,37 +14,48 @@
...
+        'screeninfo',
+        'tk',
+        'opencv-contrib-python>=4.4.0.42',
+        'opencv-python>=4.4.0.42',
+        'grpcio',
+        'grpcio-tools'],
+    packages=[package for package in find_packages() if package.startswith("diambraArena")],
+    include_package_data=True,
+    extras_require=extras,
+    classifiers=[
```
Comment (discordianfish) [inline setup.py:50]: @alexpalms Added these, lemme know if that makes sense. Here is what is available: https://pypi.org/classifiers/

**Pull request closed without merging**